lutris/util/vulkan.py

```diff
@@ -0,0 +1,19 @@
+"""Vulkan helper module"""
+import os
+from enum import Enum
+
+def vulkan_check():
+    vulkan_lib = os.path.isfile("/usr/lib/libvulkan.so")
+    vulkan_lib32 = os.path.isfile("/usr/lib32/libvulkan.so")
+    vulkan_lib_multi = os.path.isfile("/usr/lib/x86_64-linux-gnu/libvulkan.so")
```

Comment:
Note that this probably won't work on certain distributions, where the file, for instance, is actually called `/usr/lib/x86_64-linux-gnu/libvulkan.so.1`.